lighthouse-core/runner.js

```diff
@@ -149,15 +143,20 @@ class Runner {
       };
 
       // Replace ICU message references with localized strings; save replaced paths in lhr.
-      lhr.i18n.icuMessagePaths = i18n.replaceIcuMessageInstanceIds(lhr, settings.locale);
+      i18nLhr.i18n.icuMessagePaths = i18n.replaceIcuMessages(i18nLhr, settings.locale);
+
+      // LHR has now been localized.
+      const lhr = /** @type {LH.Result} */ (i18nLhr);
 
       // Create the HTML, JSON, and/or CSV string
       const report = generateReport(lhr, settings.output);
 
       return {lhr, artifacts, report};
     } catch (err) {
-      // i18n error strings
-      err.friendlyMessage = i18n.getFormatted(err.friendlyMessage, settings.locale);
+      // i18n LighthouseError strings.
+      if (err.friendlyMessage) {
```

Comment:
turns out these aren't always LHErrors and we were often formatting `undefined` :)